examples/example_brownian.py

```diff
@@ -0,0 +1,62 @@
+from parcels import Grid, ScipyParticle
+import numpy as np
+from datetime import timedelta as delta
+import math
+import random
+
+
+def two_dim_brownian_flat(particle, grid, time, dt):
+
+    # random.seed() - should a seed be included for reproducibility/testing purposes?
+    # Use equation for particle diffusion.
+    particle.lat += random.normalvariate(0, 1)*math.sqrt(2*dt*grid.K_lat)
+    particle.lon += random.normalvariate(0, 1)*math.sqrt(2*dt*grid.K_lon)
+
```

Comment:
Note that I have now created a new branch (`brownian_rng_attempt`) that has an attempt to implement a Normal distribution using the Box-Muller transform (https://en.wikipedia.org/wiki/Box%E2%80%93Muller_transform). Currently has an assertion error, but will look into that